oauth2/x_fosite_migrations_test.go

```diff
@@ -38,7 +37,7 @@ func TestXXMigrations(t *testing.T) {
 		migratest.MigrationSchemas{nil, nil, createMigrations},
 		x.CleanSQL,
 		x.CleanSQL,
-		func(t *testing.T, db *sqlx.DB, m, k, steps int) {
+		func(t *testing.T, driver string, db *sqlx.DB, m, k, steps int) {
```

Comment:
Maybe rename this to `dbName`, otherwise this would be considered the driver which would be expected to be `postgres` for CockroachDB.